stats.js

```diff
@@ -31,7 +32,18 @@ const stats = {
 
     return stats.updateDayStats(date, dayStats);
   },
-
+// pause tracking on current site
+pauseCurrentSite: async (domain) => {
+  const date = new Date().toLocaleDateString()
+  const dayStats = await stats.getDayStats(date)
+  if (!dayStats.isPaused){
+    dayStats.isPaused = []
+  }
```

Comment:
it's better to save the `isPaused` in the outer scope instead of saving it the stats of today because we don't want to lose it tomorrow.

Reply:
Also, if this is an array of domains, we should have a better variable name, `isPaused` is expected to return a boolean